sui_programmability/examples/deepbook/sources/sui.move

```diff
@@ -0,0 +1,24 @@
+module deepbook::sui {
+    use std::option::none;
+
+    use sui::coin::create_currency;
+    use sui::transfer::{public_freeze_object, public_share_object};
+    use sui::tx_context::TxContext;
+
+    const DECIMAL: u8 = 8;
+
+    struct SUI has drop {}
+
+    fun init(witness: SUI, ctx: &mut TxContext) {
+        let (treasury_cap, metadata) = create_currency<SUI>(witness, DECIMAL, b"SUI", b"SUI", b"SUI", none(), ctx);
+        public_freeze_object(metadata);
+        public_share_object(treasury_cap);
+    }
+
+    #[test_only]
+    public fun init_test(ctx: &mut TxContext) {
+        let (treasury_cap, metadata) = create_currency<SUI>(SUI {}, DECIMAL, b"SUI", b"SUI", b"SUI", none(), ctx);
+        public_freeze_object(metadata);
+        public_share_object(treasury_cap);
+    }
+}
```

Comment:
Are this and the USD files intended to be added?

Reply:
Looks like they are only intended for testing purposes. maybe it would be good to wrap the whole module as a test only or remove the init functions? Or is there some other better way to mark a whole module as test only

Reply:
Yep, you can mark a whole module as `#[test_only]` (and that would indeed make sense here).

Reply:
Done!